sdk/src/main/java/io/radar/sdk/RadarApiClient.kt

```diff
@@ -284,14 +285,40 @@ internal class RadarApiClient(
         val uri = Uri.parse(host).buildUpon()
             .appendEncodedPath("v1/track")
             .build()
-        val url = URL(uri.toString())
-
+        var url = URL(uri.toString())
         val headers = headers(publishableKey)
 
-        apiHelper.request(context, "POST", url, headers, params, true, object : RadarApiHelper.RadarApiCallback {
+        var requestParams = params
+        var replays = Radar.getReplays()
+        val replayCount = replays.size
+        var nowMS = System.currentTimeMillis()
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
+            nowMS = SystemClock.elapsedRealtimeNanos() / 1000000
+        }
+        val replaying = replayCount > 0 && options.replay == RadarTrackingOptions.RadarTrackingOptionsReplay.ALL
+        if (replaying) {
+            val replayList = JSONArray()
+            for (replay in replays) {
+                replayList.put(replay)
+            }
+            replayList.put(params)
+            requestParams = JSONObject()
+            requestParams.putOpt("replays", replayList)
+            val replayUri = Uri.parse(host).buildUpon()
+                .appendEncodedPath("v1/track/replay")
+                .build()
+            url = URL(replayUri.toString())
+        }
+
+        apiHelper.request(context, "POST", url, headers, requestParams, true, object : RadarApiHelper.RadarApiCallback {
```

Comment:
looks like the old code is missing the `logPayload` param here.

Reply:
ah maybe bc it defaulted to `true`

Reply:
it's at line 404 actually, after the callback

Reply:
ahhh got it